### PR TITLE
TestName and Grades added as separate attributes 

### PR DESCRIPTION
**Pull request by Meg1456**
We can now add a student's grades for any test.

### CODE REVIEW

File: src/seedu/addressbook/parser/Parser.java [hunk 1]
```diff
@@ -120,6 +131,9 @@ public Command parseCommand(String userInput) {
         case CreateExamCommand.COMMAND_WORD:
             return prepareCreateExam(arguments);
 
+        case AddAssignmentStatistics.COMMAND_WORD:
```
Comment (0WN463) [inline src/seedu/addressbook/parser/Parser.java]: Inconsistent use of verbs

Comment (Meg1456) [reply src/seedu/addressbook/parser/Parser.java]: Inconsistent in what way?

File: statistics.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```
Comment (micklwl) [inline statistics.txt]: Pls update .gitignore to ignore this textfile

Comment (micklwl) [inline statistics.txt]: Pls update .gitignore to ignore this txt

File: src/seedu/addressbook/parser/Parser.java (hunk 1 above)
Comment (0WN463) [reply src/seedu/addressbook/parser/Parser.java]: The command is AddAssignmentStatistics, but you prepareCreateStatistics

**Pull request merged**